design/oss_suite_release.md

```diff
@@ -76,6 +74,7 @@ In addition, as part of building this release, we'll also be creating:
   - [Jenkins](https://github.com/cyberark/conjur-credentials-plugin)
   - Kubernetes / OpenShift
     - [Kubernetes authenticator client](https://github.com/cyberark/conjur-authn-k8s-client)
+    - [Conjur Kubernetes Secrets Provider](https://github.com/cyberark/secrets-provider-for-k8s)
```

Comment:
Probably worth a discussion with IL, just so that understand this will be in the suite and what the repo requirements will be.